File: app/services/omni/shipment_detail.rb
class Omni::ShipmentDetail::Service
  include Buildit::Service::Base

  service 'ShipmentDetail', 'Omni.service'

  connected_mode(Buildit::Service::Backend::Crud) do |config|
    config.model = Omni::ShipmentDetail
  end

end # class Omni::ShipmentDetail::Service

File: app/assets/javascripts/omni/store/ShipmentDetail.js
Ext.define('Omni.store.ShipmentDetail', {
  extend       : 'Ext.data.Store',
  alias        : 'store.omni-ShipmentDetail',
  model        : 'Omni.model.ShipmentDetail',
  autoLoad     : false,
  storeId      : 'ShipmentDetailStore',
  remoteFilter : true,
  remoteSort   : true,

  constructor  : function (config) {
    var me    = this;
    me.callParent(this);
    
    var proxy = Ext.Object.merge({}, me.getProxy());
    me.setProxy(proxy);
  } // constructor

}); // Ext.define('Omni.store.ShipmentDetail'